platform/web/detect.py

```diff
@@ -203,13 +212,11 @@ def configure(env: "Environment"):
     env.Append(CPPDEFINES=["PTHREAD_NO_RENAME"])
     env.Append(CCFLAGS=["-s", "USE_PTHREADS=1"])
     env.Append(LINKFLAGS=["-s", "USE_PTHREADS=1"])
+    if cc_semver >= (3, 1, 30):
+        env.Append(LINKFLAGS=["-s", "DEFAULT_PTHREAD_STACK_SIZE=2MB"])
```

Comment:
Actually this may not need to be version specific, they just changed the default in 3.1.30 but the flag existed before, so we can always set it.